.github/workflows/helmtagbumper.yaml

```diff
@@ -61,14 +61,80 @@ jobs:
             Update versions in **${{ steps.apiprevioustag.outputs.tag }}**.
             *Note: This PR was opened by the [update-env GitHub Actions workflow](https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}).*
           labels: "datarepo,automerge,version-update"
-      - name: Slack job status
+      - name: "Notify Slack"
         if: always()
         uses: broadinstitute/action-slack@v3.8.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
         with:
           status: ${{ job.status }}
-          author_name: api_helm_bumper
-          fields: repo,message,commit,author,action,eventName,ref,workflow,job,took
+          fields: job,repo,message,author,tfook
```

Comment:
Looks like there's a small typo in `took`

Reply:
Thank you for catching this!